invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/components/CommunityHeader/CommunityHeader.js

```diff
@@ -54,6 +56,10 @@ class CommunityHeaderComponent extends Component {
                   {community.metadata.title}
                 </div>
               </>
+            ) : permissions?.can_publish_restricted ? (
```

Comment:
Thanks for your thoughtful feedback. I was trying to avoid adding yet another prop to pass, but if the team at CERN sees the benefit in that approach, we can certainly do it!
As to your observation about spacing and the use of the ternary operator, it’s a classic case of what I call "developer's haiku" - brevity may be the soul of wit, but it doesn’t always make for easily digestible code! 😉
I decided to completely remove the change in the community header component, as it adds unnecessary complexity without real significant advantages.